src/app/planner/PlannerDownloadUI.tsx

```diff
@@ -0,0 +1,24 @@
+'use client';
+
+import React from 'react';
+
+import NebulaLogo from '@/components/icons/NebulaLogo/NebulaLogo';
+import PlannerSchedule from '@/components/planner/PlannerSchedule/PlannerSchedule';
+
+
+
+export default function PlannerDownloadUI({ downloadRef } : { downloadRef : React.RefObject<HTMLDivElement | null> }) {
+  
+    return (    
+        <div className='bg-black p-4 absolute left-[-9999px]' ref={downloadRef}>
```

Comment:
Can we add `top-0` to this so it doesn't impact the page length and sticky scrolling while not visible?